airflow/providers/discord/hooks/discord_webhook.py

```diff
@@ -101,7 +101,7 @@ def _get_webhook_endpoint(self, http_conn_id: Optional[str], webhook_endpoint: O
                 'Expected Discord webhook endpoint in the form of "webhooks/{webhook.id}/{webhook.token}".'
             )
 
-        return endpoint
+        return f'https://discord.com/api/{endpoint}'
```

Comment:
I don't see this as an error since you can set your own `webhook_endpoint` in the Hook

Reply:
I mean, yes, you can, but:
1. Has to follow the regex `^webhooks/[0-9]+/[a-zA-Z0-9_-]+$`
2. It is used on the `HttpHook` as it is, it means, as an endpoint directly. With that, I mean that the URI generated will be `https://webhooks/{webhook.id}/{webhook.token}`, being it wrong.

Reply:
But what happens if you set: "https://discord.com/api/" as the "Host" in your HTTPConnection

Reply:
Hmmm, it' seems like if you do so, with the `http_con_id` is working fine. Then the problem is I guess only with the `webhook_endpoint`, which is missing the host?.

Reply:
In the operator we have instructions how to set both `http_con_id` & `webhook_endpoint` :

https://github.com/apache/airflow/blob/001dc22af9a30f710501eac85ff04297ba2a728f/airflow/providers/discord/operators/discord_webhook.py#L36-L41

Is this issue of insufficient documentation about how to set the connection or do we have a bug that either way it does not work?

Reply:
@yoshichulo ^^^ Can you check the above comment plz